merkle_drop/merkle_tree.py

```diff
@@ -21,6 +21,7 @@ def __init__(
         hash: bytes,
         *,
         parent: "Node" = None,
+        # left and right is arbitrary
```

Comment:
we can rename it to `childs = []` or so if you like that more.

Reply:
I don't know, I changed it to child1 and child2 but it did not improve it so much so I reverted it and added this comment.